package.json

```diff
@@ -4,22 +4,22 @@
   "description": "Firefox Health backend",
   "main": "build/index.js",
   "scripts": {
-    "test:src": "neutrino test",
-    "test:lint": "neutrino lint",
-    "coverage": "nyc neutrino test && nyc report --reporter=text-lcov | coveralls",
+    "test:src": "mocha",
+    "test:lint": "eslint --cache --format codeframe --ext mjs,jsx,js src test",
+    "coverage": "nyc npm run test:src && nyc report --reporter=text-lcov | coveralls",
     "test": "npm-run-all --parallel test:*",
-    "watch": "neutrino test --watch",
-    "build": "neutrino build",
-    "start": "neutrino start",
-    "start:prod": "neutrino build && node .",
-    "start:debugger": "neutrino build && node --inspect .",
+    "watch": "mocha test --watch",
+    "build": "webpack --mode production",
+    "start": "webpack --watch --mode development",
+    "start:prod": "npm run build && node .",
+    "start:debugger": "npm run build && node --inspect .",
     "precommit": "lint-staged",
     "heroku-postbuild": "npm run build",
-    "fetchNimbledroidData": "neutrino build && node build/nimbledroid.js"
+    "fetchNimbledroidData": "npm run build && node build/nimbledroid.js"
   },
   "lint-staged": {
     "*.js": [
-      "neutrino lint"
+      "npm run test:lint"
```

Comment:
Any reason for not using `yarn` instead for the package scripts? fwiw, this project uses `yarn.lock`.